cmd/containerd-shim-runhcs-v1/exec_clone.go

```diff
@@ -0,0 +1,68 @@
+package main
+
+import (
+	"context"
+
+	"github.com/Microsoft/hcsshim/internal/cmd"
+	"github.com/Microsoft/hcsshim/internal/cow"
+	"github.com/Microsoft/hcsshim/internal/log"
+	"github.com/Microsoft/hcsshim/internal/uvm"
+	"github.com/opencontainers/runtime-spec/specs-go"
+	"github.com/sirupsen/logrus"
+)
+
+func newClonedExec(
```

Comment:
I don't really like the amount of code dup between newClonedExec & newHcsExec as well as with newClonedhcsTask & newHcsTask. Could we either refactor these some or otherwise find a way to better merge them?

Reply:
As discussed earlier we can do this refactor later. But let us have an offline discussion about this.